Agreements/Internal/S29_East.txt

```diff
@@ -0,0 +1,2 @@
+COPX:*:*:POL:EGNV:*:Sector 29:East:*:20000:POL
+COPX:*:*:POL:EGNT:*:Sector 29:East:*:26000:POL
```

Comment:
240 on agreed levels

Reply:
260 is a new agreement, agreed levels to reflect soon